services/reset-demo.php

```diff
@@ -121,6 +121,39 @@
             error_log("Insert dummy data query failed: " . mysqli_error($conn));
         }
 
+        // Drop assignment table if it exists
+        $resetTableSqlAssignment = "DROP TABLE IF EXISTS assignment;";
+        $result = mysqli_query($conn, $resetTableSqlAssignment);
+        if (!$result) {
+            error_log("Drop table query failed: " . mysqli_error($conn));
+        }
+        
+        $createTableSqlAssignment = "
+        CREATE TABLE assignment (
+            course_code VARCHAR(50),
```

Comment:
I think we should have an `assignment_id` calculation as well for PRIMARY KEY. This is needed to delete the assignment by instructor in future.

Reply:
The Back end Team agreed to use the Course Code and Assignment Name Combo as primary key.